paddle/operators/recv_op.cc

```diff
@@ -161,7 +161,6 @@ class RecvOpMaker : public framework::OpProtoAndCheckerMaker {
  public:
   RecvOpMaker(OpProto *proto, OpAttrChecker *op_checker)
       : OpProtoAndCheckerMaker(proto, op_checker) {
-    AddInput("RX", "(Tensor) Input tensor to be optimized").AsDuplicable();
```

Comment:
Why this line is removed, I thought the OP need to explicitly specify its input, otherwise the framework won't know about this dependency.

Reply:
Current `recv_op` should be called `listen_and_serv`, which is often the only operator in a server program's main block (block 0). Unlike send/recv it does not appear in the calculation blocks.

Reply:
Thanks for the explanation, I think I misunderstood the code.